tests/testMaskedImage.py

```diff
@@ -127,6 +127,23 @@ def testArrays(self):
         mimage2 = afwImage.makeMaskedImageFromArrays(image, mask, variance)
         self.assertEqual(type(mimage2), type(self.mimage))
 
+    def testProperties(self):
+        self.assertImagesEqual(self.mimage.image, self.mimage.getImage())
+        self.assertImagesEqual(self.mimage.mask, self.mimage.getMask())
+        self.assertImagesEqual(self.mimage.variance, self.mimage.getVariance())
+        image2 = self.mimage.image.Factory(self.mimage.getDimensions())
+        image2.array = 5.0
+        self.mimage.image = image2
+        self.assertImagesEqual(self.mimage.image, image2)
+        mask2 = self.mimage.mask.Factory(self.mimage.getDimensions())
+        mask2.array = 0x4
+        self.mimage.mask = mask2
+        self.assertMasksEqual(self.mimage.mask, mask2)
+        var2 = self.mimage.image.Factory(self.mimage.getDimensions())
+        var2.array = 3.0
+        self.mimage.variance = var2
+        self.assertImagesEqual(self.mimage.variance, var2)
+
```

Comment:
I've now done that in `testImage.py`, but I don't think we need to do it here as well, as `MaskedImage` doesn't actually have a `.array` interface (it's all delegated to `Image` and `Mask`).